locale/en/LC_MESSAGES/django.po

```diff
@@ -4648,12 +4640,12 @@ msgstr "Type(s) of research"
 msgid ""
 "\n"
 "                    De FETC-GW onderscheidt 3 typen onderzoek (die evt. ook "
-"samen in één studie voor kunnen komen):\n"
+"samen in één aanvraag voor kunnen komen):\n"
 "                "
 msgstr ""
 "\n"
 "    The FEtC-H distinguishes 3 types of research (which may all be used "
-"within one study):\n"
+"within one proposal):\n"
 "    "
```

Comment:
Ik denk niet dat aanvraag/proposal/application hier echt op zijn plek is? Zeker aangezien op andere plekken wordt gesproken over 'onderzoek' in dezelfde context. Wat ik ook beter vind passen eigenlijk, aangezien we hier meer abstract over onderzoeksvormen praten en niet over een specifieke aanvraag.

Reply:
Ja hier had ik hem beter als study kunnen laten